Project/src/MakeCall/AddParticipantPopover.js

```diff
@@ -43,9 +48,19 @@ export default function AddParticipantPopover(props) {
                             <h3 className="add-participant-panel-header">Add a participant</h3>
                             <div className="add-participant-panel-header">
                                 <TextField className="text-left" label="Identifier" onChange={e => setUserId(e.target.value)} />
-                                <TextField className="text-left" label="Alternate Caller Id (For adding phone number only)" onChange={e => setAlternateCallerId(e.target.value)} />
-                                <Button className="mt-3" onClick={handleAddCommunicationUser}>Add CommunicationUser</Button>
+                                { 
+                                    call._kind === CallKind.TeamsCall && 
+                                    <TextField className="text-left" label="Thread Id (Needed if SDK is initialized for Teams User )" onChange={e => setThreadId(e.target.value)} />
+                                }
+                                {
+                                    call._kind === CallKind.Call && 
+                                    <TextField className="text-left" label="Alternate Caller Id (For adding phone number only)" onChange={e => setAlternateCallerId(e.target.value)} />
+                                }
+                                <Button className="mt-3" onClick={handleAddParticipant}>Add Participant</Button>
+                                {
+                                    call._kind === CallKind.Call && 
```

Comment:
the indentation between here and line below is wrong and makes it difficult to read. We should really have prettier running over these repos :/

Reply:
I will create a different PR to run prettier